### DISCUSSION TITLE
Enlaces sobre manejo de github en Equipo

### DISCUSSION THREAD

**Discussion by LauTIC2022** [LuisFelipeCaro/Ecommerce_tech#18] [Ideas]
Hola compañeros, estaba revisando lo de github Desktop y les iba a enviar mi propio video, pero me encontré con esta lista que me parece mucho más útil para nuestro trabajo. Revísenla y  me cuentan ;). Es de un tocayo de Luis Felipe jejeje.

Utilizando Github Desktop en equipo-Luis Felipe 
**https://youtu.be/KwsGxVTCIg0**

**Comment by LuisFelipeCaro** [comment]
Muchas gracias 👍
